src/mozilla_sec_eia/models/sec10k/notebooks/exhibit21_extractor.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<p>Maybe would add this: "Model finetuning will only be run if configured to do so (<code>layoutlm_training_run = None</code>&nbsp;) , otherwise a pretrained version will be used from the&nbsp;<code>mlflow</code>&nbsp;tracking server.</p><p>Also I think in step 2 we should say Named Entity Recognition (NER) since it's the first place we use NER.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/mozilla-sec-eia/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>